docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,58 @@
+# Release 362 (XX Sep 2021)
+
+## General
+
+* Add support for {func}`LISTAGG`. ({issue}`4835`)
```

Comment:
Lowercase `listagg` to enable a reference. 
<img width="374" alt="Screen Shot 2021-09-13 at 0 54 09" src="https://user-images.githubusercontent.com/6237050/132994348-c9672f6d-e7e9-40f1-a16f-879817059095.png">
<img width="353" alt="Screen Shot 2021-09-13 at 0 54 43" src="https://user-images.githubusercontent.com/6237050/132994350-4c7d3556-3c78-45e5-a086-336ee9de51ad.png">